akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -178,6 +180,20 @@ object FSM {
     } // so we do not have to break source/binary compat.
     // TODO: Can be removed once we can break State#timeout signature to `Option[Duration]`
 
+    /**
+     * JAVA API
+     * u
```

Comment:
runaway `u`, remove. Also, make it `Java API:` and then have the text directly without linefeed or else some of the doc views will only show "Java API" and no actual description.